resources/user-agents/browsers/puffin/puffin-5-0-on.json

```diff
@@ -101,24 +101,28 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*IP*",
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*I?*",
           "devices": {
             "iPhone": "iPhone",
             "iPad": "iPad"
           },
           "engine": "WebKit",
           "platforms": [
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2",
-            "iOS_A"
+            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A",
+            "iOS_C_10_0", "iOS_C_10_1", "iOS_C_10_2", "iOS_C_10_3",
+            "iOS_C"
           ]
         },
         {
           "match": "Mozilla/5.0 (#PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/*Safari/* Puffin/#MAJORVER#.#MINORVER#.*I?*",
           "device": "general Mobile Device (Apple)",
           "engine": "WebKit",
           "platforms": [
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2",
-            "iOS_A"
+            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A",
+            "iOS_C_10_0", "iOS_C_10_1", "iOS_C_10_2", "iOS_C_10_3",
```

Comment:
iOS_C_8_0 is coded without the zero, iOS_C_9_0 too. Because of this iOS 8 and 9 must have their Zero version at last. iOS_C_10_0 is coded with the Zero, buit we should keep the ordering like in the other places.
Maybe we should change the ordering for iOS_A in a separate PR too.